Update smokey references (#4925)
alphagov/govuk-developer-docs: source/manual/pingdom.html.md

GOV.UK uses [Pingdom](https://www.pingdom.com/) to monitor the external availability of our services, integrating with [PagerDuty](/manual/pagerduty.html) to notify developers if a service becomes unavailable (owing to a DNS issue or network misconfiguration, for example).

> Our [end-to-end tests](https://github.com/alphagov/govuk-e2e-tests), is not as reliable at detecting problems like these, as it runs on the same local network as our services, and thus can't necessarily guarantee for us that users can reach our services.

Reviewer: ```suggestion
Our [end-to-end tests](https://github.com/alphagov/govuk-e2e-tests) are not as reliable at detecting problems like these, as they run on the same local network as our services, and thus can't necessarily guarantee for us that users can reach our services.
```
